pkg/ccl/importccl/pg_testdata_helpers_test.go

```diff
@@ -92,6 +92,13 @@ func getMultiTablePostgresDumpTestdata(t *testing.T) string {
 			})
 		})
 	}
+	_ = os.Remove(dest + ".bz2")
+	out, err := exec.Command("bzip2", "-k", dest).CombinedOutput()
+	if err != nil {
+		t.Fatal(err, string(out))
+	}
+	gzipFile(t, dest)
```

Comment:
Do these files need to be generated, checked in, and tested too?

Reply:
it wouldn't hurt, but in both cases the code is outside the format-specific block.

Reply:
oh, I meant to remove this part of the diff.

Reply:
Fixed.